2021/07/30 21:30:32 Starting App Insight Logger for task:  runTaskLet
2021/07/30 21:30:32 Version: 3.0.01664.0001 Branch: 2021-07-23 Commit: 3e1c760
2021/07/30 21:30:32 Attempt 1 of http call to http://10.0.0.5:16384/sendlogstoartifacts/info
2021/07/30 21:30:32 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:30:32 Response code 500, retrying after 1 seconds
[2021-07-30T21:30:33.030714] Entering context manager injector.
[2021-07-30T21:30:33.562286] context_manager_injector.py Command line Options: Namespace(inject=['ProjectPythonPath:context_managers.ProjectPythonPath', 'RunHistory:context_managers.RunHistory', 'TrackUserError:context_managers.TrackUserError'], invocation=['train.py', '--C', '0.8480321322688669', '--max_iter', '75'])
Script type = None
[2021-07-30T21:30:33.566208] Entering Run History Context Manager.
2021/07/30 21:30:33 Attempt 2 of http call to http://10.0.0.5:16384/sendlogstoartifacts/info
2021/07/30 21:30:33 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:30:33 Response code 500, retrying after 2 seconds
[2021-07-30T21:30:34.301292] Current directory: /mnt/batch/tasks/shared/LS_root/jobs/quick-starts-ws-153057/azureml/hd_0341301e-da13-4d2f-afc9-465a815bcd0f_4/wd/azureml/HD_0341301e-da13-4d2f-afc9-465a815bcd0f_4
[2021-07-30T21:30:34.301482] Preparing to call script [train.py] with arguments:['--C', '0.8480321322688669', '--max_iter', '75']
[2021-07-30T21:30:34.301499] After variable expansion, calling script [train.py] with arguments:['--C', '0.8480321322688669', '--max_iter', '75']

2021/07/30 21:30:35 Attempt 3 of http call to http://10.0.0.5:16384/sendlogstoartifacts/info
2021/07/30 21:30:35 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:30:35 Response code 500, retrying after 4 seconds
2021/07/30 21:30:37 Not exporting to RunHistory as the exporter is either stopped or there is no data.
Stopped: false
OriginalData: 1
FilteredData: 0.
2021/07/30 21:30:39 Attempt 4 of http call to http://10.0.0.5:16384/sendlogstoartifacts/info
2021/07/30 21:30:39 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:30:39 Response code 500, retrying after 8 seconds
/azureml-envs/azureml_ba9520bf386d662001eeb9523395794e/lib/python3.6/site-packages/sklearn/linear_model/logistic.py:433: FutureWarning: Default solver will be changed to 'lbfgs' in 0.22. Specify a solver to silence this warning.
  FutureWarning)
/azureml-envs/azureml_ba9520bf386d662001eeb9523395794e/lib/python3.6/site-packages/sklearn/linear_model/logistic.py:460: FutureWarning: Default multi_class will be changed to 'auto' in 0.22. Specify the multi_class option to silence this warning.
  "this warning.", FutureWarning)


[2021-07-30T21:30:40.398828] The experiment completed successfully. Finalizing run...
Cleaning up all outstanding Run operations, waiting 900.0 seconds
2 items cleaning up...
Cleanup took 0.3955686092376709 seconds
[2021-07-30T21:30:41.416015] Finished context manager injector.
2021/07/30 21:30:47 Attempt 5 of http call to http://10.0.0.5:16384/sendlogstoartifacts/info
2021/07/30 21:30:47 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:30:47 Final http request try failed with error: <nil>
. Request: POST /sendlogstoartifacts/info HTTP/1.0
Host: 10.0.0.5:16384
Content-Type: application/json
User-Agent: AmlCompute-Hosttools/linux/3.0.01664.0001-3e1c760

{"process_name":"worker","node_id":"tvmps_75183f59b018ce35141bfb6d01e5722dd06a7ed00a3a50e85166280567e34537_d","pid_in_container":86,"private_ip":"10.0.0.5","log":"azureml-logs/70_driver_log.txt"}

2021/07/30 21:30:47 Request max attempts reached. End of retries.
2021/07/30 21:30:47 Retries exhausted. Http call failed, Url: http://10.0.0.5:16384/sendlogstoartifacts/info
2021/07/30 21:30:47 Fail to send to master with error: InternalError - :{
Info: <nil>
}
 
2021/07/30 21:30:47 Failed to send process info logs to master server with err: InternalError - :{
Info: <nil>
}
2021/07/30 21:30:47 Attempt 1 of http call to http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:30:47 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:30:47 Response code 500, retrying after 1 seconds
2021/07/30 21:30:48 Attempt 2 of http call to http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:30:48 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:30:48 Response code 500, retrying after 2 seconds
2021/07/30 21:30:50 Attempt 3 of http call to http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:30:50 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:30:50 Response code 500, retrying after 4 seconds
2021/07/30 21:30:54 Attempt 4 of http call to http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:30:54 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:30:54 Response code 500, retrying after 8 seconds
2021/07/30 21:31:02 Attempt 5 of http call to http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:31:02 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:31:02 Final http request try failed with error: <nil>
. Request: POST /sendlogstoartifacts/status HTTP/1.0
Host: 10.0.0.5:16384
Content-Type: application/json
User-Agent: AmlCompute-Hosttools/linux/3.0.01664.0001-3e1c760

{"process_name":"worker","timestamp":"2021-07-30T21:30:47.976640037Z","status":"Started"}

2021/07/30 21:31:02 Request max attempts reached. End of retries.
2021/07/30 21:31:02 Retries exhausted. Http call failed, Url: http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:31:02 Fail to send to master with error: InternalError - :{
Info: <nil>
}
 
2021/07/30 21:31:02 Failed to send process info logs to master server with err: InternalError - :{
Info: <nil>
}
2021/07/30 21:31:02 Attempt 1 of http call to http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:31:02 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:31:02 Response code 500, retrying after 1 seconds
2021/07/30 21:31:03 Attempt 2 of http call to http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:31:03 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:31:03 Response code 500, retrying after 2 seconds
2021/07/30 21:31:05 Attempt 3 of http call to http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:31:05 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:31:05 Response code 500, retrying after 4 seconds
2021/07/30 21:31:07 Not exporting to RunHistory as the exporter is either stopped or there is no data.
Stopped: false
OriginalData: 1
FilteredData: 0.
2021/07/30 21:31:09 Attempt 4 of http call to http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:31:09 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:31:09 Response code 500, retrying after 8 seconds
2021/07/30 21:31:17 Attempt 5 of http call to http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:31:17 Http request failed with status: 500
Response body: Unable to write to log.


2021/07/30 21:31:17 Final http request try failed with error: <nil>
. Request: POST /sendlogstoartifacts/status HTTP/1.0
Host: 10.0.0.5:16384
Content-Type: application/json
User-Agent: AmlCompute-Hosttools/linux/3.0.01664.0001-3e1c760

{"process_name":"worker","timestamp":"2021-07-30T21:31:02.983713231Z","status":"Completed"}

2021/07/30 21:31:17 Request max attempts reached. End of retries.
2021/07/30 21:31:17 Retries exhausted. Http call failed, Url: http://10.0.0.5:16384/sendlogstoartifacts/status
2021/07/30 21:31:17 Fail to send to master with error: InternalError - :{
Info: <nil>
}
 
2021/07/30 21:31:17 Failed to send process info logs to master server with err: InternalError - :{
Info: <nil>
}
2021/07/30 21:31:17 Not exporting to RunHistory as the exporter is either stopped or there is no data.
Stopped: false
OriginalData: 1
FilteredData: 0.
2021/07/30 21:31:17 Process Exiting with Code:  0
2021/07/30 21:31:18 All App Insights Logs was sent successfully or the close timeout of 20 was reached
